public/app/features/dashboard/services/DashboardLoaderSrv.ts

```diff
@@ -111,8 +111,14 @@ export class DashboardLoaderSrv {
       return Promise.reject('expecting path parameter');
     }
 
+    const queryString: { [key: string]: any } = {};
+
+    params.forEach((value, key) => {
+      queryString[key] = value;
```

Comment:
should we exclude 'path'?

Reply:
we could, path is already assumed since the handler will know the path. Not sure it'd harm anything in including though. up to you!